docs-site/content/27.0/api/vector-search.md

```diff
@@ -2190,6 +2190,8 @@ The text used to generate embeddings for the `embedding` field will be `passage:
 
 Once you've indexed your embeddings in a vector field, you can now search for documents that are "closest" to a given query vector. 
 
+#### Pagination
```

Comment:
Instead of adding a section heading here, let's add a tip block below the code snippets, that explain pagination. We can even re-use the same text and example you have in the guide to make it clear.

Reply:
> Instead of adding a section heading here, let's add a tip block below the code snippets, that explain pagination. 

Refactored in https://github.com/typesense/typesense-website/pull/259/commits/e8040faf145d4ed009d1bba8478854840e79b053